tests/integration/project/cmd_project_test.go

```diff
@@ -49,6 +49,17 @@ var _ = Describe("odo project command tests", func() {
 		})
 	})
 
+	Context("odo machine readable output on get project", func() {
+		It("should be able to get project", func() {
+			projectListJSON := helper.CmdShouldPass("odo", "project", "get", "-o", "json")
+			listOutputJSON, err := helper.Unindented(projectListJSON)
+			Expect(err).Should(BeNil())
+			partOfProjectListJSON, err := helper.Unindented(`{"kind":"Project","apiVersion":"odo.dev/v1alpha1","metadata":{"name":"` + project + `","namespace":"` + project + `","creationTimestamp":null},"spec":{},"status":{"active":true}}`)
```

Comment:
How is it a part of json? I think we are mentioning the whole json here

```
$ odo project get -o json
{
        "kind": "Project",
        "apiVersion": "odo.dev/v1alpha1",
        "metadata": {
                "name": "default",
                "namespace": "default",
                "creationTimestamp": null
        },
        "spec": {},
        "status": {
                "active": true
        }
}
```

Reply:
ok, will update